index.bs

```diff
@@ -2618,27 +2637,27 @@ credential|credentials=]. The [=client=] and user will then use whichever is ava
     };
 </xmp>
 
-This enumeration's values describe the [=[RP]=]'s requirements for [=resident credentials=]:
+This enumeration's values describe the [=[RP]=]'s requirements for [=client-side discoverable credentials=] (formerly known as [=resident credentials=] or [=resident keys=]):
 
 <div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
     :   <dfn>discouraged</dfn>
-    ::  This value indicates the [=[RP]=] prefers creating a [=non-resident credential=], but will accept a
-        [=resident credential=].
+    ::  This value indicates the [=[RP]=] prefers creating a [=server-side credential=], but will accept a
+        [=client-side discoverable credential=].
 
     :   <dfn>preferred</dfn>
-    ::  This value indicates the [=[RP]=] prefers creating a [=resident credential=], but will accept a
-        [=non-resident credential=].
+    ::  This value indicates the [=[RP]=] prefers creating a [=client-side discoverable credential=], but will accept a
+        [=server-side credential=].
 
     :   <dfn>required</dfn>
-    ::  This value indicates the [=[RP]=] requires a [=resident credential=], and is prepared to receive an error
-        if a [=resident credential=] cannot be created.
+    ::  This value indicates the [=[RP]=] requires a [=client-side discoverable credential=], and is prepared to receive an error
+        if a [=client-side discoverable credential=] cannot be created.
 </div>
 
-Note: The [=[RP]=] can use a combination of the value provided for |options|.{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{residentKey}}
-and the [=credProps|Credential Properties Extension=]'s return value to determine  whether or not the authenticator created a [=resident credential=].
-This is useful when values of  {{ResidentKeyRequirement/discouraged}} or {{ResidentKeyRequirement/preferred}} are used for
-|options|.{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{residentKey}}, because in those cases it is possible for an
-[=authenticator=] to create either a [=resident credential=] or a [=non-resident credential=].
+Note: The [=[RP]=] can use a combination of the values provided for <code>|options|.{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{residentKey}}</code>
```

Comment:
There's only one value named before the introduction of a new noun (the credprops return value), right?
```suggestion
Note: The [=[RP]=] can use a combination of the value provided for <code>|options|.{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{residentKey}}</code>
```

Reply:
Well, "values" here refers to the following multiple things, but this clause is overall poor, so I'll polish it, thx for the heads-up.

Reply:
pls see 02348ab